test/Microsoft.AspNet.Mvc.TagHelpers.Test/TestModelMetadataProvider.cs

```diff
@@ -0,0 +1,184 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using Microsoft.AspNet.Mvc.ModelBinding.Metadata;
+using Microsoft.Framework.Internal;
+using Xunit;
+
+namespace Microsoft.AspNet.Mvc.ModelBinding
+{
+    public class TestModelMetadataProvider : DefaultModelMetadataProvider
```

Comment:
suggest using `compileFiles: [ "../Microsoft.AspNet.Mvc.Core.Test/TestModelMetadataProvider.cs" ]` in project.json for other projects that need this